ui/widgets/nuxeo-directory-suggestion.js

```diff
@@ -23,6 +23,7 @@ import '@nuxeo/nuxeo-elements/nuxeo-element.js';
 import { FormatBehavior } from '../nuxeo-format-behavior.js';
 import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
 import './nuxeo-selectivity.js';
+import { escapeHTML } from './nuxeo-selectivity.js';
```

Comment:
This causes a lint issue. see the PR check https://qa.nuxeo.org/jenkins/job/Other/job/nuxeo-elements-pr-check/243/console